From 15aabe0f6f7cf5ff2f221e39797ea007361cc369 Mon Sep 17 00:00:00 2001
From: Don Zickus <dzickus@redhat.com>
Date: Tue, 16 Sep 2014 17:10:27 -0400
Subject: [usb] pl2303: remove bogus delta_msr_wait wake up

Message-id: <1410887463-20674-556-git-send-email-dzickus@redhat.com>
Patchwork-id: 95535
O-Subject: [RHEL7 PATCH 555/591] USB: pl2303: remove bogus delta_msr_wait wake up
Bugzilla: 1110939
RH-Acked-by: Tony Camuso <tcamuso@redhat.com>
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>

https://bugzilla.redhat.com/show_bug.cgi?id=1110939

commit 71c671bf4cead0e801eccbed3fe790e817f6b6d4
Author: Johan Hovold <jhovold@gmail.com>
Date:   Sun Dec 29 19:22:59 2013 +0100

    USB: pl2303: remove bogus delta_msr_wait wake up

    Remove bogus MSR wait-queue wake up from process_read_urb which never
    updates the MSR flags.

    Signed-off-by: Johan Hovold <jhovold@gmail.com>
    Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/drivers/usb/serial/pl2303.c b/drivers/usb/serial/pl2303.c
index 5ef25d1..bec01fb 100644
--- a/drivers/usb/serial/pl2303.c
+++ b/drivers/usb/serial/pl2303.c
@@ -937,7 +937,6 @@ static void pl2303_process_read_urb(struct urb *urb)
  line_status = priv->line_status;
  priv->line_status &= ~UART_STATE_TRANSIENT_MASK;
  spin_unlock_irqrestore(&priv->lock, flags);
- wake_up_interruptible(&port->port.delta_msr_wait);
 
  if (!urb->actual_length)
   return;
-- 
1.7.1